virtusize/src/main/java/com/virtusize/android/VirtusizeImpl.kt

```diff
@@ -136,30 +157,17 @@ internal class VirtusizeImpl(
                         }
                     }
 
-                    VirtusizeEvents.UserLoggedIn.getEventName() -> {
-                        // Updates the user session and fetches updated user products and body profile from the server
+                    is VirtusizeEvent.UserClosedWidget ->
                         scope.launch {
                             virtusizeRepository.updateUserSession()
-                            virtusizeRepository.fetchDataForInPageRecommendation()
-                            virtusizeRepository.updateInPageRecommendation()
                         }
```

Comment:
Updating user session when the widget is closed fixed https://app.clickup.com/t/3702259/NSDK-240

Reply:
Understood. Thanks.